docs/content/modeling/modular-models.mdx

```diff
@@ -24,18 +24,17 @@ import {
 
 <DocumentationNotice />
 
-Modular models allows splitting your authorization module across multiple files and modules, improving upon some of the challenges that may be faced when operating an authorization model within a company, such as:
+Modular models allow you to split your authorization module across multiple files and modules. 
 
-- A model can grow large and difficult to understand
-- As more teams begin to contribute to a model, the ownership boundaries may not be clear and code review processes might not scale
+As authorization models grow larger and more difficult to understand as more teams contribute to a model. As a result, model ownership boundaries may not be clear and code review processes might not scale.
```

Comment:
Maybe a lead in to this might help?

```suggestion
As adoption of <ProductName format={ProductNameFormat.ShortForm}/> increases, authorization models can grow larger and more difficult to understand. As a result, model ownership boundaries may not be clear and code review processes might not scale.
```